api/setup.cfg

```diff
@@ -35,7 +35,7 @@ combine_as_imports=True
 line_length=88
 
 [tool:pytest]
-addopts = --reuse-db --randomly-seed=1521188766 --randomly-dont-reorganize
+addopts = --reuse-db --randomly-seed=1521188766 --randomly-dont-reorganize -W=default
```

Comment:
The `-W` option does not require the `=` to assign the value and the docs suggest to do it without. It's still valid though ^^

Reply:
Pytest uses `Argparse` internally. Argparse always also allows for using the equal sign. I prefer this, as IMO it's easier to parse with the eyes and in my experience it's more universally supported (but that's subjective). I also wanted to be consistent with `--randomly-seed=1521188766`.